.github/workflows/editorconfig.yml

```diff
@@ -0,0 +1,20 @@
+name: editorconfig
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  editorconfig:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: technote-space/get-diff-action@v1.2.8
+    - name: fetch image
+      run: docker pull mstruebing/editorconfig-checker
+    - name: editorconfig check
+      run: |
+        docker run --volume="$PWD":/check mstruebing/editorconfig-checker ec \
```

Comment:
By the way, is this one faster?

```
VERSION="2.0.4"
OS="linux"
ARCH="amd64"
curl -O -L -C - https://github.com/editorconfig-checker/editorconfig-checker/releases/download/$VERSION/ec-$OS-$ARCH.tar.gz && \
tar xzf ec-$OS-$ARCH.tar.gz && \
./bin/ec-$OS-$ARCH -disable-indentation ${{ env.GIT_DIFF }}
```

I have not checked what the container actually adds as a benefit.

Reply:
Yeah, it probably is faster. I picked the container so it doesn't need to be bumped to pick up new versions.

Reply:
I actually see it as a feature if it does not update itself. It could be easy enough to add though:


```
VERSION=$(curl https://api.github.com/repos/editorconfig-checker/editorconfig-checker/releases/latest | jq -r '.name')
curl -O -L -C - https://github.com/editorconfig-checker/editorconfig-checker/releases/download/$VERSION/ec-linux-amd64.tar.gz && \
tar xzf ec-linux-amd64.tar.gz && \
./bin/ec-linux-amd64 -disable-indentation ${{ env.GIT_DIFF }}
```

Reply:
> I actually see it as a feature if it does not update itself.

Yeah good point, sticking to a tag would be safer considering how busy this repo is.

I'll send a PR.